### PR TITLE
Add translation packaging

### PR DESCRIPTION
**Pull request by jpetrell**

### CODE REVIEW

File: daemon/daemon.pro [hunk 1]
```diff
@@ -26,5 +26,31 @@ SOURCES += \
...
 include($$PWD/CryptoImpl/CryptoImpl.pri)
 
+# translations
+TS_FILE = $$OUT_PWD/sailfish-secrets.ts
+EE_QM = $$OUT_PWD/sailfish-secrets_eng_en.qm
+
+ts.commands += lupdate $$PWD -ts $$TS_FILE
+ts.CONFIG += no_check_exist
+ts.output = $$TS_FILE
+ts.input = .
```
Comment (rainemak) [inline daemon/daemon.pro]: This could also be $$PWD

File: rpm/sailfish-secrets.spec [hunk 2]
```diff
@@ -230,6 +241,7 @@ cp -R lib/Crypto/doc/html/* %{buildroot}/%{_docdir}/Sailfish/Crypto/
 %files -n sailfishsecretsdaemon
 %defattr(-,root,root,-)
 %{_bindir}/sailfishsecretsd
+/usr/share/translations/sailfish-secrets_eng_en.qm
```
Comment (rainemak) [inline rpm/sailfish-secrets.spec]: %{_datadir}/translations ...

**Comment by rainemak** [comment]
Added minor comments, cleanup and merge. LGTM

File: rpm/sailfish-secrets.spec (hunk 2 above)
Comment (Venemo) [inline rpm/sailfish-secrets.spec]: Why not use `%{_datadir}` here too?

Comment (jpetrell) [reply rpm/sailfish-secrets.spec]: Copy&paste issue. Thanks fixed.

**Pull request merged**